app/domain/authentication/authn_jwt/validate_and_decode/validate_and_decode_token.rb

```diff
@@ -24,10 +24,10 @@ def call
           validate_signature
           fetch_jwt_claims_to_validate
           validate_claims
-          decoded_token_after_claims_validation
+          decode_and_validate_token_with_claims
```

Comment:
do we need to have this call? the `validate_claims` method already calls it.

Reply:
Removed